lib/computed_model.rb

```diff
@@ -311,18 +311,27 @@ def computing_plan(deps)
   # @return [Hash{Symbol=>Array}]
   def self.normalize_dependencies(deps)
```

Comment:
Please update the yard annotation above:

```diff
-  # @param deps [Array<Symbol, Hash>]
+  # @param deps [Array<Symbol, Hash>, Hash]
```

Reply:
fixed e86934f